client/src/components/Masthead/Masthead.component.tsx

```diff
@@ -43,7 +43,7 @@ const Masthead: FC = () => {
     <Box bg="secondary.200" px={{ base: '24px', sm: '29px', xl: '33px' }}>
       <Flex direction="row">
         <Center my={mastheadTopMarginY}>
-          <Image src={LionHeadSymbol} />
+          <Image src={LionHeadSymbol} htmlHeight="15.2px" htmlWidth="12.8px" />
```

Comment:
Surely it should be..
```suggestion
          <Image src={LionHeadSymbol} boxSize="20px" />
```

Reply:
i tried this at first, but it was still giving that same "Image elements do not have explicit width and height" warning

Reply:
okay, but I would have thought the height and width of the image should be 20px, not the dimensions you specified

Reply:
went with 15.2px for both height and width, referred to the [Design System doc](https://www.figma.com/file/yATTXTZ5Goy9XDy5qooSO3/AskGov-Design-System?node-id=2142%3A24)